packages/channels/botpress-channel-rocketchat/README.md

```diff
@@ -0,0 +1,149 @@
+# botpress-channel-rocketchat
+
+![botpress-rocketchat](https://github.com/RocketChat/botpress-channel-rocketchat/wiki/images/botpress.gif)
+
+Official Facebook Rocket.Chat connector module for [Botpress](http://github.com/botpress/botpress).
+
+This module has been build to accelerate and facilitate development of Rocket.Chat bots.
+
+## Installation
+
+To install this module just run:
+
+```sh
+npm install botpress-channel-rocketchat
+```
+
+## Get started
+
+To setup connexion of your chatbot to Rocket.Chat follow this steps:
+
+* If you are in a development environment up a docker instance of Rocket.Chat,
+see how to make it in our example repository:
+
+[botpress-kick-starter](https://github.com/RocketChat/botpress-kick-starter)
+
+* Login with your user;
+
+* Create a new user for your bot;
+
+* Add your bot to wanted channels;
+
+## Features
+
+### Incoming
+
+* [Profile](#profile)
+* [Text messages](#text-messages)
+
+### Outgoing
+
+* [Text messages](#text-messages-1)
+
+## Reference
+
+### Incoming
+
+You can listen to incoming event easily with Botpress by using `bp` built-in `hear` function. You only need to listen to specific Rocket.Chat event to be able to react to user's actions.
+
+```js
+bp.hear({platform:'rocketchat', type: 'message', text:'hello'}, async (event, next) => {
+  await bp.rocketchat.sendText(event.channel, 'Hi I\'m alive', {})
```

Comment:
Content in Botpress X is sent using renderers, i.e. we don't want to expose these `send...` functions anymore

Reply:
Hi @slvnperron , first thank you very much for all this revision. I solved all the other problems, and I am starting to solve this one now. I searched on the other channels and didn't find an example of exposing data using renderers, do you have an example of code or documentation in how to develop this?